src/main.js

```diff
@@ -109,97 +122,97 @@ var quotes = [
   "Each person must live their life as a model for others.",
   "A champion is defined not by their wins but by how they can recover when they fall."
 ];
-var savedPosters = [
-  makePoster("./assets/pug.jpg", "Optimism", "Keep a joyful heart!")
-];
+
+var savedPosters = [];
 var currentPoster;
 
-window.onload = displayRandomPoster;
-savePosterBtn.addEventListener('click', saveCurrentPoster);
-showRandomBtn.addEventListener('click', displayRandomPoster);
-showFormBtn.addEventListener('click', displayForm);
-makePosterBtn.addEventListener('click', createNewPoster);
-showSavedBtn.addEventListener('click', displaySavedPosters);
-showMainBtn.addEventListener('click', displayMain);
-backToMainBtn.addEventListener('click', displayMain);
+// event listeners go here 👇
 
-function displayRandomPoster() {
-  var randomImage = images[getRandomIndex(images)];
-  var randomTitle = titles[getRandomIndex(titles)];
-  var randomQuote = quotes[getRandomIndex(quotes)];
+window.addEventListener('load', showPoster);
 
-  currentPoster = makePoster(randomImage, randomTitle, randomQuote);
+savePosterButton.addEventListener('click', savePoster);
+randomButton.addEventListener('click', showPoster);
+makeYourOwnPosterButton.addEventListener('click', openForm);
+showSavedPostersButton.addEventListener('click', displayOnGrid);
+backToMainButton.addEventListener('click', hideSavedPostersForm);
+//event listeners on make your own poster page
+showMyPosterButton.addEventListener('click', showMyPoster);
+takeMeBackButton.addEventListener('click', hideForm);
 
-  displayCurrent();
-}
+// functions and event handlers go here 👇 (we've provided one for you to get you started)!
 
+// functions on main generate random poster page
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
-}
-
-function makePoster(imageURL, title, quote) {
-  return new Poster(imageURL, title, quote);
-}
-
-function createNewPoster(event) {
+};
+
+function generateRandomPoster() {
+  currentPoster = new Poster(images[getRandomIndex(images)],
+  titles[getRandomIndex(titles)],
+  quotes[getRandomIndex(quotes)]);
+};
+
+function showPoster() {
+  generateRandomPoster();
+  posterImage.src = currentPoster.imageURL;
+  posterTitle.innerText = currentPoster.title;
+  posterQuote.innerText = currentPoster.quote;
+};
```

Comment:
It looks like `showPoster` may be doing more than it should. It's generating a new poster, but also setting the attributes to show the poster. Could `showPoster` take `imageURL`, `title` and `quote` as parameters? If so, the function might be able to be reused in another part of the app